rskj-core/src/main/resources/reference.conf

```diff
@@ -229,7 +229,7 @@ transaction.gasPriceBump = 40
 transaction.accountSlots = 16
 
 # flag to enable rate-limit for accounts broadcasting transactions consuming large amounts of resources
-transaction.accountTxRateLimit.enabled = false
+transaction.accountTxRateLimit.enabled = true
```

Comment:
If the feature is enabled in `reference.conf`, it will be enabled by default for all networks unless otherwise specified. Therefore you don't need to enable it anywhere else other than here.

Reply:
Perfect-o